code/game/machinery/vending_types.dm

```diff
@@ -77,28 +77,61 @@
 		/obj/item/reagent_containers/food/drinks/bottle/bluecuracao = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/kahlua = 5,
 		/obj/item/reagent_containers/food/drinks/bottle/triplesec = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/sarezhiwine = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/champagne = 5,
 		/obj/item/reagent_containers/food/drinks/bottle/vodka = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/vodka/mushroom = 1,
 		/obj/item/reagent_containers/food/drinks/bottle/pulque = 5,
 		/obj/item/reagent_containers/food/drinks/bottle/fireball = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/whiskey = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/victorygin = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/makgeolli = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/soju = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/sake = 1,
 		/obj/item/reagent_containers/food/drinks/bottle/cremewhite = 4,
 		/obj/item/reagent_containers/food/drinks/bottle/mintsyrup = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/nmshaan_liquor = 2,
 		/obj/item/reagent_containers/food/drinks/bottle/chartreuseyellow = 5,
 		/obj/item/reagent_containers/food/drinks/bottle/messa_mead = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/small/ale = 6,
+		/obj/item/reagent_containers/food/drinks/bottle/small/beer = 6,
+		/obj/item/reagent_containers/food/drinks/bottle/small/skrellbeerdyn = 10,
+		/obj/item/reagent_containers/food/drinks/bottle/small/xuizijuice = 8,
+		/obj/item/reagent_containers/food/drinks/bottle/small/khlibnyz = 4,
+		/obj/item/reagent_containers/food/drinks/bottle/cola = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/space_mountain_wind = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/space_up = 5,
+		/obj/item/reagent_containers/food/drinks/bottle/hrozamal_soda = 2,
+		/obj/item/reagent_containers/food/drinks/bottle/small/midynhr_water = 3,
+		/obj/item/reagent_containers/food/drinks/bottle/dominian_wine = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/assunzione_wine = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/algae_wine = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/kvass = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/tarasun = 1,
+		/obj/item/reagent_containers/food/drinks/bottle/valokki_wine = 1,
```

Comment:
Remove the bottle of valokki wine in the bartender's booze closet if you're gonna do this.

Reply:
Already removed